automation/project/Dependencies.scala

```diff
@@ -23,6 +23,7 @@ object Dependencies {
     "com.fasterxml.jackson.core" % "jackson-databind" % jacksonV,
     "com.fasterxml.jackson.core" % "jackson-core" % jacksonV,
     "com.fasterxml.jackson.module" % "jackson-module-scala_2.11" % jacksonV,
+    "net.virtual-void" %% "json-lenses" % "0.6.2" % "test",
```

Comment:
Had to look up what json lenses were - that's pretty cool :)

Reply:
Ah, yes. I previously used this in "unit" tests in the other PR for this feature.